codegen.yml

```diff
@@ -1,6 +1,6 @@
 overwrite: true
-schema: "https://subquery-dev.webb.tools/graphql"
-documents: 
+schema: "https://standalone-subql.webb.tools/graphql"
+documents:
   - "./packages/page-statistics/**/*.graphql"
```

Comment:
There should be no files in `packages/*`. Anything related to stats-dapp should live under `apps/`